docs/specifications/category-3d-tiles.md

```diff
@@ -10,9 +10,9 @@ The 3D Tiles category can represent the major tiled 3D formats:
 
 | Loader                                                                     | Notes |
 | -------------------------------------------------------------------------- | ----- |
-| [`Tiles3DLoader`](modules/3d-tiles/docs/api-reference/tiles-3d-loader)     |       |
-| [`CesiumIonLoader`](modules/3d-tiles/docs/api-reference/cesium-ion-loader) |       |
-| [`I3SLoader`](modules/i3s/docs/api-reference/i3s-loader)                   |       |
+| [`Tiles3DLoader`](/docs/modules/3d-tiles/api-reference/tiles-3d-loader)     |       |
+| [`CesiumIonLoader`](/docs/modules/3d-tiles/api-reference/cesium-ion-loader) |       |
+| [`I3SLoader`](/docs/modules/i3s/api-reference/i3s-loader)                   |       |
```

Comment:
You can update the potree link to match the format.

Reply:
There is only README.md in potree folder
Could the potree's api-reference be in some other place?

Reply:
I never wrote that page since the potree loader isn't finished. That is why I suggest we might create a new placeholder page.